src/components/AnonymousChat/child/ChatItemText.style.ts

```diff
@@ -128,11 +134,11 @@ export const styles = StyleSheet.create({
     borderRadius: 20,
     backgroundColor: colors.lightgrey
   },
-  ml8: {
-    marginLeft: dimen.normalizeDimen(8)
+  mlBuble: {
+    marginLeft: dimen.normalizeDimen(BUBBLE_LEFT_PADDING)
   },
-  mr8: {
-    marginRight: dimen.normalizeDimen(8)
+  mrBuble: {
+    marginRight: dimen.normalizeDimen(BUBBLE_RIGHT_PADDING)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change here seems to be renaming the style properties for better readability and understanding. It's a good practice to have meaningful names for variables and properties. However, make sure these renamed properties are updated everywhere in the codebase where they are used.
```diff
-  ml8: {
+  mlBuble: {
     marginLeft: dimen.normalizeDimen(BUBBLE_LEFT_PADDING)
   },
-  mr8: {
+  mrBuble: {
     marginRight: dimen.normalizeDimen(BUBBLE_RIGHT_PADDING)
   },
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->